multihost_test/Bugzillas/test_regression.py

```diff
@@ -5,6 +5,12 @@
 import pytest
 
 
+def exceute_cmd(multihost, command):
```

Comment:
s/exceute_cmd/execute_cmd

Reply:
By the way, why don't you reuse the `execute_cmd` function provided [conftest.py](https://github.com/RedHatQE/shadow-utils/blob/main/multihost_test/Bugzillas/conftest.py#L30), instead of providing yet another implementation?